ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
+    openshift-release-golang-1.20:
+      as:
+      - golang:1.20
```

Comment:
```suggestion
```

Seems that we can [simply get rid of this](https://github.com/openshift/release/blob/master/ci-operator/config/quay/quay-operator/quay-quay-operator-master__ocp-latest.yaml#L16-L18) :smile_cat: 
@Amoghrd can we please give it a try ?

Reply:
will do